components/molecule/autosuggest/src/components/MultipleSelection.js

```diff
@@ -86,6 +86,7 @@ const MoleculeAutosuggestFieldMultiSelection = ({
         disabled={disabled}
         required={required}
         tabIndex={tabIndex}
+        autoComplete="off"
```

Comment:
Use "nope" instead if you want to force to no autosuggest https://developer.mozilla.org/es/docs/Web/Security/Securing_your_site/desactivar_autocompletado_formulario